app/views/layouts/cms.html.erb

```diff
@@ -12,12 +12,6 @@
     <%= stylesheet_link_tag    'application', '/assets/css/app.css?a=1499756908' %>
     <%= stylesheet_link_tag    'search_auto_complete' %>
 
-    <%= javascript_include_tag 'application', 'https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js'%>
-    <%= javascript_include_tag 'application', 'https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js'%>
-
-    <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-    <%= javascript_include_tag 'search_auto_complete' %>
-
     <%= csrf_meta_tags %>
```

Comment:
Moved to ensure that the libraries load in the correct order for the auto complete to work.